pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -1932,9 +1932,13 @@ func TestPreFilterDisabled(t *testing.T) {
 	nodeInfo := framework.NewNodeInfo()
 	node := v1.Node{}
 	nodeInfo.SetNode(&node)
-	p := &InterPodAffinity{}
+	ctx := context.Background()
+	n := func(plArgs runtime.Object, fh framework.Handle) (framework.Plugin, error) {
+		return New(plArgs, fh, feature.Features{})
+	}
+	p := plugintesting.SetupPlugin(ctx, t, n, &config.InterPodAffinityArgs{}, cache.NewEmptySnapshot(), namespaces)
```

Comment:
Uhm... `SetupPlugin` would start informers that we don't need in this test (potentially making the test slower).
Either just use `New` or add a `SetupPlugin` variant that doesn't have informers.

Reply:
Done.

Reply:
shouldn't this test use the variant with no informer?

Same as below